internal/config_loader/validator.go

```diff
@@ -398,18 +399,10 @@ func (v *Validator) validateCELExpression(expr string, path string) {
 	// Clean up the expression (remove leading/trailing whitespace and newlines)
 	expr = strings.TrimSpace(expr)
 
-	// Try to parse the expression
-	ast, issues := v.celEnv.Parse(expr)
+	// Syntax validation only
+	_, issues := v.celEnv.Parse(expr)
 	if issues != nil && issues.Err() != nil {
 		v.errors.Add(path, fmt.Sprintf("CEL parse error: %v", issues.Err()))
-		return
-	}
-
-	// Try to check the expression (type checking)
-	// Note: This may fail for dynamic variables, which is acceptable
-	_, issues = v.celEnv.Check(ast)
```

Comment:
The value types are using dynamic types. It's meaningless to check the type now as we won't fail if the type is not matched.